client/src/style/_config.scss

```diff
@@ -20,6 +20,9 @@ $faded-modal-text-color: #7d7d7d;
 $normal-text-color: black;
 $white-text-color: white;
 $section-header-text-color: #bbbbbb;
+$tooltip-background-color: #efefef;
```

Comment:
Is it possible to keep the `tooltip` colours in `_config.scss` and delete `Tooltip.scss` and `Tooltip.tsx`?

Reply:
These diffs might disappear if you rebase `ImageList` on `main` because ToolTip was merged to `main`

Reply:
Lol i was so confused how those appeared I didnt realize that you rebased 😆